src/lib/transport.js

```diff
@@ -37,17 +40,24 @@ function createPacketAndCondition ({
   const address = destinationAccount + '.~' + protocol + '.' + receiverId +
     (id ? ('.' + id) : '')
```

Comment:
Since we're already using the payment `id` to ensure that the hashes are different every time, it seems like we might as well use that as a nonce in the generation of the encryption key as well. 

I think we should move the id out of the ILP address and into the transport `unsafeHeaders` section. That ID isn't used for routing and isn't useful for anyone aside from the sender and receiver.

@justmoon thoughts?